op-chain-ops/cmd/op-upgrade/main.go

```diff
@@ -123,6 +123,11 @@ func entrypoint(ctx *cli.Context) error {
 	}
 
 	chainIDs := ctx.Uint64Slice("chain-ids")
+	if len(chainIDs) != 1 {
+		// This requirement is due to the `SYSTEM_CONFIG_START_BLOCK` environment variable
+		// that we read from in `op-chain-ops/upgrades/l1.go`
+		panic("op-upgrade currently only supports upgrading a single chain at a time")
```

Comment:
I think we can return an error here instead of panicking.

Reply:
Either is fine, we panic elsewhere with this sort of issue, either way the invariant must hold that the program does not execute when the case is encountered, both can do that, i lean towards not needing to do another round of diffs and staying consistent with the other panics in this codebase

Reply:
Either is fine, we panic elsewhere with this sort of issue, either way the invariant must hold that the program does not execute when the case is encountered, both can do that, i lean towards not needing to do another round of diffs and staying consistent with the other panics in this codebase